ledger-api/grpc-definitions/com/digitalasset/ledger/api/v1/value.proto

```diff
@@ -44,7 +44,7 @@ message Value {
     // epoch is greater than that. Range: 0001-01-01T00:00:00Z to
     // 9999-12-31T23:59:59.999999Z, so that we can convert to/from
     // https://www.ietf.org/rfc/rfc3339.txt
-    sfixed64 timestamp = 9;
+    sfixed64 timestamp = 9 [jstype = JS_STRING];
```

Comment:
Why not use the `[jstype = JS_STRING]` annotation for `int32 date = 14;` below as well? Dates seem to be converted to strings manually in `mapping/value.ts`, whereas timestamps are converted to strings using the annotation.

Reply:
`[jstype = JS_STRING]` only works on 64-bit numeric types.